--- mozilla-unified/gfx/layers/d3d11/genshaders.py.old	2023-10-07 15:37:52.084397991 -0700
+++ mozilla-unified/gfx/layers/d3d11/genshaders.py	2023-10-07 16:24:42.124594895 -0700
@@ -100,7 +100,7 @@
         proc_stdout = subprocess.check_output(argv)
         proc_stdout = decode_console_text(sys.stdout, proc_stdout)
         deps = find_dependencies(proc_stdout)
-        assert "fxc2" in fxc_location or len(deps) > 0
+        assert "fxc" in fxc_location or len(deps) > 0
 
         with open(temp_filename, "r") as temp_fp:
             output_fp.write(temp_fp.read())
